README.md

```diff
@@ -54,34 +98,24 @@ This section provides a general introduction on how to start using the app.
     cd llm_app/
     pip install --upgrade --extra-index-url https://packages.pathway.com/966431ef6ba -r requirements.txt
     ```
-### Usage
 
-1. **Environment Variables:** Create an .env file in `llm_app/` directory and add the following environment variables, adjusting their values according to your specific requirements and setup.
-    ```bash
-    PATHWAY_REST_CONNECTOR_HOST=127.0.0.1
-    PATHWAY_REST_CONNECTOR_PORT=8080
-    OPENAI_API_TOKEN=<Your Token>
-    PATHWAY_CACHE_DIR=/tmp/cache
-    ```
-    If you are not running the local version, please remember to replace `OPENAI_API_TOKEN` with your personal API token, which you can generate from your account on [openai.com](https://platform.openai.com/account/api-keys).
-
-2. **Run the App:** You can start the application with the command:
+- **Run the App:** You can start the application with the command:
     ```bash 
     cd llm_app/
-    python main.py --mode contextful
+    python main.py
     ```
-    The `--mode contextful` option means that the software will index the documents in `data/pathway-docs` and take them into account when processing queries. The pathway pipeline being run is in `llm_app/pathway_pipelines/contextful/app.py`
 
-    You can also run the app without the need for external APIs by using `local` mode.
+### Usage
 
-3. **Send REST queries** (in a separate terminal window): These are examples of how to interact with the application once it's running. `curl` is a command-line tool used to send data using various network protocols. Here, it's being used to send HTTP requests to the application.
+1. **Send REST queries** (in a separate terminal window): These are examples of how to interact with the application once it's running. `curl` is a command-line tool used to send data using various network protocols. Here, it's being used to send HTTP requests to the application.
     ```bash
     curl --data '{"user": "user", "query": "How to connect to Kafka in Pathway?"}' http://localhost:8080/ | jq
 
     curl --data '{"user": "user", "query": "How to use LLMs in Pathway?"}' http://localhost:8080/ | jq
     ```
+    Please change `localhost` to `0.0.0.0` if you are running the app on docker.
 
-4. **Test reactivity by adding a new file:** This shows how to test the application's ability to react to changes in data by adding a new file and sending a query.
+2. **Test reactivity by adding a new file:** This shows how to test the application's ability to react to changes in data by adding a new file and sending a query.
```

Comment:
It won't work in docker. It'd be better to add a data volume than put in a Dockerfile. Putting it in Dockerfile gives users less flexibility for experimentation.